spark/src/main/resources/spark.yml

```diff
@@ -41,3 +41,16 @@ scalar_functions:
       - args:
           - value: i64
         return: DECIMAL<P,S>
+  - name: shift_right
+    description: >-
+      Bitwise (signed) shift right.
+      Params:
+        base – the base number to shift.
+        shift – number of bits to right shift.
+    impls:
```

Comment:
It looks like in Spark the base can be either int or long, and return type is set accordingly. I  we should add both options?

Quickly testing this, it works for `select shiftright(col, 2) from (values (bigint(1234)) as table(col))` but not for `select shiftright(col, 2) from (values (1234) as table(col))`, so yep I think we need to list both versions here.

Reply:
Overall I think it's fine to add the function here initially, but it'd be good to also file the PR/Issue on the core functions since this seems general enough, I think :)

Reply:
I agree, although that might take a bit longer ;)